docs/questions/native-editor/filter-widgets.md

```diff
@@ -0,0 +1,110 @@
+
+### Dropdown menu and search
+
+To create a dropdown menu with search and a list of all values:
+
+1. Include a variable in your query.
+2. Set the **Variable type** to **Field filter**. If the query lacks a database field, you could use a Text or Number type as well, depending on what you're filtering.
+3. Set **Field to map to** to the appropriate field (only if you selected the field filter variable type).
+4. Set **Filter widget operator** to whatever [operator](#filter-widget-operators) you want.
+5. Set **How should users filter on this variable** to "Dropdown list". If you're not using a field filter, you'll need to edit the dropdown list settings to [tell Metabase where to get the values to list in the dropdown](#customizing-values-for-dropdown-lists-and-search-boxes).
+
+If there are too many different values in that column to display in a dropdown menu, Metabase will simply display a search box instead. So if you have a lot of email addresses, you may just get a search box anyway. The dropdown menu widgets work better when there's a small set of values to choose from (like the fifty U.S. states).
+
+## Customizing values for dropdown lists and search boxes
+
+When you add a dropdown menu or search box, you can tell Metabase which values people can choose from when using a filter with a dropdown list or search box.
+
+1. Add a dropdown list or search box.
+2. Next to the option you chose, click **Edit**.
+3. Metabase will pop up a modal where you can select **Where the values should come from**.
+
+You can choose:
+
+- **From connected fields**. If you selected the Field filter variable type, you'll also have the option to use the connected field.
+- **From another model or question**. If you select this option, you'll need to pick a model or question, then a field from that model or question that Metabase will use to supply the values for that dropdown or search box. For example, if you want the dropdown to list the different plans an account could be on, you could select an "Account" model you created, and select the field "Plan" to power that dropdown. The dropdown would then list all of the distinct plan options that appear in the "Plan" column in the Accounts model.
+- **Custom list**. Enter each item on a line. You can enter any string values you like.
+
+You can also [change a dashboard filter's selectable values](../../dashboards/filters.md#change-a-filters-selectable-values).
+
+## Requiring a value for a filter widget
+
+In the **Variable** settings sidebar, you can toggle the **Always require a value** option. If you turn this on:
+
+- You must enter a default value.
+- The default value will override any [optional syntax](./sql-parameters.md#making-variables-optional) in your code (like an optional `WHERE` clause). If no value is passed to the filter, Metabase will run the query using the default value. Click on the **Eye** icon in the editor to preview the SQL Metabase will run.
```

Comment:
The section for default value is in a different doc, but it feels like it should with be in this doc or at least linked in this doc